docs/configuration/module-config/serial.mdx

```diff
@@ -294,3 +294,170 @@ previousState = currentState; // update the previous state
 
 }
 ```
+### Interfacing with button an relay With External Microcontroller
```

Comment:
an ->and
Maybe this instead:
Interfacing a Button and Relay with an External Microcontroller